src/fileInfo.js

```diff
@@ -86,12 +86,20 @@ fileInfo.prototype.getContentType = function () {
 
 /**
  * Gets last modified time of file/folder
- * @returns {integer}   Last modified time of file/folder
+ * @returns {Date}   Last modified time of file/folder
  */
 fileInfo.prototype.getLastModified = function () {
   return new Date(this.fileInfo['{DAV:}getlastmodified'])
 }
 
+/**
+ * Gets arbitrary property
+ * @returns {string}   Value of the property
```

Comment:
and `@param {string} property Name of the property for which to retrieve the value`

also in general I wonder whether we had JSDoc setup?

Reply:
https://owncloud.github.io/js-owncloud-client/